htdocs/contrat/class/api_contracts.class.php

```diff
@@ -436,8 +436,81 @@ public function putLine($id, $lineid, $request_data = null)
 		$updateRes = $this->contractLine->update(DolibarrApiAccess::$user);
 
 		if ($updateRes > 0) {
-			$result = $this->get($id);
-			unset($result->line);
+			if (getDolGlobalInt('API_CONTRAT_PUTLINE_OUTPUT_LINE_ONLY')) {
+				$result = $this->contractLine;
+				$result->fetch($id);
```

Comment:
```suggestion
				$result->fetch($lineid);
```
grosse bêtise de ma part